lib/util/splice.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Plain old regular Array#splice may cause stackoverflow if we have too many items
+ */
+function chunkedSplice(array, start, deleteCount, items) {
+  var chunkSize = 10000
+  var splice = [].splice
```

Comment:
(1) …use it for this one?

Some of the code is written in a funky way as it improves gzip a bunch, having these constant methods has helped shave of bytes

Reply:
fixed